img/m/meltwater.com.svg

```diff
@@ -0,0 +1 @@
-744c257.269-409.24 536.808-784.11 536.808-784.11s103.96 117.7 494.23 736.35l489.79 776.45 144.11 9.35 144.09 9.36V27.545l-191.79 9.116-191.81 9.18-15.33 842.846-15.31 842.853-365.99-574.38-365.94-574.399h-258.99L762.516 1173.42l-378.994 590.55-8.135-866.726-8.16-866.705H0Zm14574.1-155.79c0 62.47-145.6 226.83-247.6 279.65-195.7 101.14-471.8 23.48-603.8-169.86-38-55.67-69-112.88-69-127.3 0-14.33 207-26.06 460.1-26.06 377.7 0 460.3 7.76 460.3 43.57zM13898.8 28.522c-245 74.428-432.9 243.34-548.5 492.737-153.6 331.614-84.6 782.391 159.1 1039.481 223.9 235.87 666.1 324.51 958 191.89 261.5-118.78 462.5-424.6 496.7-755.552l17.4-168.878h-1366.3l19.5-97.095c45.6-227.624 302.3-423.103 555.8-423.103 135 0 344.5 82.957 410.3 162.466 60.9 73.518 102.3 63.684 212.8-50.336l98.2-101.32-102.3-99.043c-145.2-140.782-375.3-220.616-626.9-217.36-113.8 1.434-241.4 13.155-283.6 26.113ZM12502.5 41.87c-126.6 56.063-280.9 238.848-319.8 378.912-17.7 63.815-32.3 326.542-32.3 583.857l-.2 467.94h-245.4v306.82h245.4v490.87h368.2V1779.4h552.3v-306.82h-552.3V980.214c0-481.68 1.6-493.856 75.3-567.57 61.7-61.795 105.8-75.34 244.9-75.34 184.1 0 194.6-11.591 210.2-230.121l6.5-92.076-230.1-7.097c-140.4-4.233-266.3 8.857-322.7 33.86zm-1524.7 353.583c212.4 108.353 224.4 137.592 214.6 525.074l-8.7 346.012-114.8 92.84c-87.6 70.72-154.4 95.99-282.3 106.64-92.2 7.65-202.5 5.12-245.3-5.58-108.8-27.25-294.7-198.17-344-316.22-87.4-209.074-25-520.161 131.8-657.343 178.3-156.214 447.6-194.178 648.7-91.423zM10386 35.944c-352.1 131.34-594.86 491.437-597.6 886.479-2.15 314.267 214 667.547 493.8 806.937 227.6 113.51 566.6 115.2 775.9 3.98 173.5-92.32 171.7-92.13 171.7-15.32 0 53.92 20.4 61.38 168.6 61.38h168.8V30.539h-168.8c-148.3 0-168.7 7.425-168.7 61.342 0 76.25-.4 76.25-160.1-7.555-151.1-79.183-529.3-105.944-683.6-48.382ZM6151.84 50.79c-126.19 66.485-226.48 161.619-285.98 271.145-39.1 72.086-52.43 206.877-61.43 621.378l-11.55 529.166h-239.36v306.92h245.41v490.87h368.23V1779.4h552.29v-306.82h-552.29V980.214c0-481.68 1.61-493.856 75.31-567.57 61.43-61.469 105.92-75.34 241.75-75.34h166.54l35.16-139.675c19.34-76.771 27.09-152.762 17.19-168.718-30.09-48.705-448.93-32.1-551.27 21.88zM4172.7 1158.1c-.34 60.2-132.58 229.41-222.07 283.93-111.4 67.91-326.13 81.19-431.84 26.67-100.41-51.62-205.54-154.5-254.82-249.31l-43.92-84.36h476.39c261.99 0 476.3 10.33 476.26 23.07zM3451.81 35.359c-257.13 95.721-496.52 350.717-566.7 603.318-81.37 292.987-4.13 673.933 182.42 900.163 277.65 336.57 884.99 367.81 1207.83 62.33 136.94-129.58 241.16-363.68 258.58-580.61l15.42-192.36H3186.82l20.85-68.986c51.63-171.173 124.75-280.019 238.96-355.621 102.97-68.176 157.61-81.657 330.25-81.657 184.45 0 222.3 11.006 350.27 101.126l143.48 101.127 104.49-79.572c57.47-43.89 104.49-97.349 104.49-118.838 0-55.608-115.19-154.001-276.14-235.984-159.49-81.134-595.09-112.782-751.74-54.436h.08" clip-path="url(#a)" style="fill:#2bb8d3;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.20476 0 0 -.20476 -735.11 1451.471)"/></svg>
```

Comment:
There is quite a lot of invisible SVG "residue" in your version of the icon. Here's a cleaner one:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1865.4 774.7"><path fill="#2bb8d3" d="M1237.6 770c0-1.6 25.3-20 56.3-40.8 78.7-53 155-118.2 236-201.9 85.5-88.4 116-125.7 116-142 0-26.3-160.7-195.5-264-277.7a1639.7 1639.7 0 0 0-103.3-76L1235 3.5l139.9-1.7L1514.7 0l48.3 39.4c109 88.8 201.1 188.4 269.8 291.5 40.5 61 41.3 54-14.1 131.1a1385.6 1385.6 0 0 1-269.6 281.1l-38.2 29.8h-136.7c-75.1 0-136.6-1.2-136.6-2.9zm-219-245.3c82-57.7 101.5-154.3 47-233.3-36.2-52.7-111.5-81.7-170.5-65.8-122.5 33-167 177.7-82.8 269.3 59.4 55.8 149.3 69.4 206.2 29.8zm-148 242c-87.3-15.7-146.3-47.3-212-113.3C521 514.8 509.2 306.4 630 148 680.4 81.7 749.8 36.8 839.7 12c92-25.2 205-8.3 290.4 43.3 79 47.8 136.6 119.3 168.8 209.8 14 39.3 16.4 56.6 16.5 118.3.1 62.8-2.2 78.8-17.3 122.5-30.2 87.5-95.1 169.1-168.8 212.3-69.5 40.7-183.4 62-258.8 48.4zm-569.3-35.6C203 653.6 92.5 533 21.8 425.7-1.8 389.9-2 389 7.7 368.7c33.3-70 196.3-252.2 297.8-333L350.3 0l139.1 1.7 139.1 1.7L579.7 36a1484 1484 0 0 0-221.8 184c-81.2 81-138 149.2-138 165.5 0 17.4 56 83.3 145.5 171.2C447.3 637.2 509 688.4 583 737.6c24.9 16.6 45.2 31.3 45.2 32.8 0 1.4-61.6 2.6-136.8 2.6H354.6z"/></svg>
```

Reply:
Any idea why that is? I run it through SVGOMG with the default settings. Are there some settings that should be tweaked?

Reply:
There's a whole "Meltwater" lettering below the logo, but it's clipped out of the visible portion (`clipPath`). I don't think SVGOMG will touch that, even though it's invisible and won't affect the display - so there's nothing much you could have done there.

Just due to the suspiciously large file size I opened it in "inkscape", had a look and removed the clipping - and here we are. :wink:

Reply:
Oh, well that was my mistake, I didn't realize that clipping in Inkscape actually doesn't remove it 😅 How should one crop the svg in a situation like this? I wasn't able to find a svg with only the icon, so I thought I could DIY it myself by removing the text 😅

Reply:
Thanks, I'll try that next time!